hack/lima-kubestellar.yaml

```diff
@@ -0,0 +1,179 @@
+  arch: "x86_64"
+  digest: "sha256:b63f266fa4bdf146dea5b0938fceac694cb3393688fb12a048ba2fc72e7bfe1b"
+- location: "https://cloud-images.ubuntu.com/releases/24.04/release-20241119/ubuntu-24.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+  digest: "sha256:6e1f90d3e81b90202b46c3573590867e575e504af2c63dd5c9b529f174e3d793"
+# Fallback to the latest release image.
+# Hint: run `limactl prune` to invalidate the cache
+- location: "https://cloud-images.ubuntu.com/releases/24.04/release/ubuntu-24.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+- location: "https://cloud-images.ubuntu.com/releases/24.04/release/ubuntu-24.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+
+mounts:
+- location: "~"
+- location: "/tmp/lima"
+  writable: true
+# containerd is managed by Docker, not by Lima, so the values are set to false here.
+containerd:
+  system: false
+  user: false
+provision:
+- mode: system
+  # This script defines the host.docker.internal hostname when hostResolver is disabled.
+  # It is also needed for lima 0.8.2 and earlier, which does not support hostResolver.hosts.
+  # Names defined in /etc/hosts inside the VM are not resolved inside containers when
+  # using the hostResolver; use hostResolver.hosts instead (requires lima 0.8.3 or later).
+  script: |
+    #!/bin/sh
+    sed -i 's/host.lima.internal.*/host.lima.internal host.docker.internal/' /etc/hosts
+- mode: system
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    command -v docker >/dev/null 2>&1 && exit 0
+    export DEBIAN_FRONTEND=noninteractive
+    curl -fsSL https://get.docker.com | sh
+    # NOTE: you may remove the lines below, if you prefer to use rootful docker, not rootless
+    systemctl disable --now docker
+    apt-get install -y uidmap dbus-user-session
+    echo 'fs.inotify.max_user_watches = 524288' >> /etc/sysctl.conf
+    echo 'fs.inotify.max_user_instances = 512' >> /etc/sysctl.conf
+    sysctl --system
+    snap install yq 
+    snap install kubectl --classic
+    snap install helm --classic
+    curl -s https://ocm.software/install.sh | sudo bash
+
+    # curl -s https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
+    wget -q https://raw.githubusercontent.com/kubestellar/kubeflex/main/scripts/install-kubeflex.sh
+    chmod +x install-kubeflex.sh
+    ./install-kubeflex.sh --ensure-folder /usr/local/bin --strip-bin
+- mode: user
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    systemctl --user start dbus
+    dockerd-rootless-setuptool.sh install
+    docker context use rootless
+
+    echo "Adding alias for kubectl..."
+    echo "alias k=kubectl" >> ~/.bashrc
+    source ~/.bashrc
+  
+    echo "Installing Kind..."
+    KIND_ARCH=$(uname -m)
+    case "$KIND_ARCH" in
+      x86_64) KIND_ARCH="amd64" ;;
+      aarch64) KIND_ARCH="arm64" ;;
+    esac
+
+    curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.20.0/kind-linux-${KIND_ARCH}
+    chmod +x ./kind
+    sudo mv ./kind /usr/local/bin/kind
```

Comment:
Why do a `sudo` in the user section instead of doing this install in the system section?